ReactWindows/ReactNative/Views/Image/ReactImageManager.cs

```diff
@@ -278,54 +267,31 @@ private void OnImageStatusUpdate(Border view, ImageLoadStatus status, ImageMetad
         private async void SetUriFromSingleSource(Border view, string source)
         {
             var imageBrush = (ImageBrush)view.Background;
-            var tag = view.GetTag();
-
-            var disposable = default(SerialDisposable);
-            if (!_disposables.TryGetValue(tag, out disposable))
+            OnImageStatusUpdate(view, ImageLoadStatus.OnLoadStart, default(ImageMetadata));
+            try
             {
-                disposable = new SerialDisposable();
-                _disposables.Add(tag, disposable);
-            }
-
-            if (BitmapImageHelpers.IsBase64Uri(source))
-            {
-                var image = new BitmapImage();
-
-                disposable.Disposable = image.GetStreamLoadObservable().Subscribe(
-                    status => OnImageStatusUpdate(view, status.LoadStatus, status.Metadata),
-                    _ => OnImageFailed(view));
-
-                using (var stream = await BitmapImageHelpers.GetStreamAsync(source))
-                {
-                    await image.SetSourceAsync(stream);
-                }
+                var imagePipeline = ImagePipelineFactory.Instance.GetImagePipeline();
+                var image = default(BitmapSource);
+                var uri = new Uri(source);
 
-                imageBrush.ImageSource = image;
-            }
-            else if (BitmapImageHelpers.IsHttpUri(source))
-            {
-                OnImageStatusUpdate(view, ImageLoadStatus.OnLoadStart, default(ImageMetadata));
-                try
+                // Base64 or local images
+                if (source.StartsWith("data:") || source.StartsWith("ms-appx:"))
```

Comment:
Yeah, it's supported in Fresco but I'll need to make some small modification that works for most cases. Thanks for telling me.